### ISSUE TITLE
10.7 browserhax impossible to do in the way its described in guide

### ISSUE THREAD

**Issue by Nalorokk** [Plailect/Guide#684]
Tried to do it in the way described in guide, took more than 4 hours in tries to do so. Anytime "please update your system" message appared faster than browserhax happened.

There is solution which allows block this message on 10.7 - you need to add rule on your router that will DROP packets to conntest.nintendowifi.net.

Something like iptables -A FORWARD -d 69.25.139.140 -j DROP.

Then there will be no "please upgrade your system" message in browser and it will easily do exploit. But for some reason it will can't download payload, so user should download ropbin payload and rename it to "browserhax_hblauncher_ropbin_payload.bin". 

Also menuhax will not connect to its servers without removing that line from router. So while in homebrew menu you should remove that line and then proceed like in guide.

Also for some impossible reason you link wrong page for 10.7 "https://yls8.mtheall.com/3dsbrowserhax.php" while it must be "http://yls8.mtheall.com/3dsbrowserhax_auto.php"

While it may took days to do browserhax in that variant that described on site, my variant works from first try. I did same downgrade twice.

**Comment by pbanj** [comment]
the page it links to shows the qr code to scan. you scan it with the 3ds camera instead of typing in the url

**Comment by ekids1234** [comment]
hello, can you illustrate your way more detailed ? 
Setting DNS is an out of date way , I want to know other way to assist, not just depend on fast ... 
( I am doubting if 10.7 could enter HBL by browser "now"...)

**Comment by cyruslee0421** [comment]
Tried to DROP 69.25.139.140, but still not working. 
Now, there are two cases after dropping the package:
1. Load for a long time and show the connection is failed.
2. Load for a long time and ask for update again

**Comment by Nalorokk** [comment]
[Here](https://www.reddit.com/r/3dshacks/comments/5eenq5/super_mario_3d_land_3ds_successfully_downgraded/) is confirmation from someone who used this method for downgrade

**Comment by gerardozamudio** [comment]
I can confirm this worked for me. My ISP provided modem/router doesn't support firewall rules so I ended up tethering to my phone and installing AFWall+ on it. Cyanogenmod 12.1 (Android 5.1.1). Go to Menu > Set Custom Script and type in 

`$IPTABLES -A FORWARD -D 69.25.139.140 -j DROP`

Select Enable Firewall and launch the browser. If you already had the update nag:

1. Format 3DS
2. Set up an internet connection in the 3DS first without firewall rule
3. Launch browser, set up bookmark
4. Shut down 3DS
5. Add firewall rule to router/Android
6. Boot 3DS
7. Launch browser, go to bookmark

It took a couple of tries because it kept freezing on the green screen but eventually Homebrew Launcher loaded. 

![Screenshot of firewall rule in AFWall+](https://cloud.githubusercontent.com/assets/1297669/20616160/d550e022-b2a5-11e6-9a7e-73d13ca9faee.png)

**Comment by TCattd** [comment]
TL;DR: I wasted 3 hours trying to get the browserhax to load before reading this issue.
Blocked conntest.nintendowifi.net with iptables at router level, and (with a small modification from the guide) browserhax loaded instantly at first try.
Need to be aware of a small change afterwards about the ropbin. Keep reading for details.

I can confirm that blocking conntest.nintendowifi.net at router level let the browserhax work at first try. Tested with a New 3DS (Mario 3D Land bundle) on 10.7.0-32U.

First i formatted the 3DS. Setup the internet connection during the initialization. Connected the 3DS to and old router running DD-WRT.

After the console initialization, i setup the url block on the router. On DD-WRT: Administration -> Commands -> paste the line and save as Firewall Script.

```
iptables -I FORWARD -d conntest.nintendowifi.net -j DROP
```

Reboot the router.
(For some reason, blocking the IP directly didn't do it for me. The 3DS still showed the update nag at the browser. But blocking conntest.nintendowifi.net with iptables worked fine).

Now you can follow @Plailect guide, but with a small modification: the "browserhax_hblauncher_ropbin_payload.bin" copy at the root of the SD card as @Nalorokk stated at the beginning of this issue.

I QR code to the browserhax URL in the guide, if that matters (didn't added it as a favorite). The homebrew launcher started immediately.

Then follow Plailect guide as usual. Still with the iptables block running at the router.

I got a partial downgrade. The process crash. So i needed to run the process another time. Again QR code to the browserhax URL (menuhax was lost in the downgrade, didn't bother to install it again), but the homebrew launcher didn't load after a grey screen.

What i needed to do was to download a new ropbin for the new firmware version (the partial downgrade version), 9.2.0. That's not told in the guide anywhere btw @Plailect . So i downloaded the 9.2.0 ropbin and replaced the browserhax_hblauncher_ropbin_payload.bin in the sdcard root with it.
QR code browserhax again, and the homebrew launcher load without issues at first try.

The downgrade completed this time, no issues. Follow the guide again.

I removed the iptables block from the router just before we need to perform the System Update. Removed the line from DD-WRT, restarted the router, and proceed to update the New 3DS. No problems.

Completed the guide like that without issues. Everything up and running.

**Comment by ihcsankml** [comment]
i'm struggle trying this guide, currently i used new 3ds LL (japanese) 10.7.0.32. I used all method described aboves, using router also my android phone. The browser updatenag didn't showed up, BUT the browser didn't load the browserhax page (i used & tried many links). It kept loading for long time and then time out error. I already put the ropbin in root sd and rename it as you suggest aboves.

If you know the solution please tell me.
Sorry for my bad english

**Comment by Plailect** [comment]
Does not seem to work for Japanese region devices, unknown as to why.

**Comment by Xeptum0117** [comment]
After struggling for a while trying to load browserhax on a 10.7.0-32U Mario 3D Land New 3ds I came across this post, and I'm somehow more confused than I was before. How can I add the suggested lines to router? I don't have an android phone and don't know how to do it from my PC. Any help would be really appreciated!

**Comment by kaicherry** [comment]
Has someone tried/considered the *inverse* for JPN units? That is, block everything *but* the browserhax sight?
